### PR TITLE
add https

### PR DESCRIPTION
**Pull request by QuentinPerez**
#4

### CODE REVIEW

File: patches/usr/local/bin/configure_ssl.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ ! -d /etc/apache2/ssl ];
+then
+	mkdir -p /etc/apache2/ssl
+fi
+
+if [ ! -f /etc/apache2/ssl/apache.crt -o ! -f /etc/apache2/ssl/apache.key ];
+then
+	(echo FR; echo France; echo Paris; echo Scaleway; echo Scaleway Technology; echo $(/usr/local/bin/oc-metadata --cached PUBLIC_IP_ADDRESS); echo ;) | openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/apache2/ssl/apache.key -out /etc/apache2/ssl/apache.crt 2> /dev/null > /dev/null
```
Comment (moul) [inline patches/usr/local/bin/configure_ssl.sh]: remove space between redirection and output: `> /dev/null` -> `>/dev/null`

Comment (moul) [inline patches/usr/local/bin/configure_ssl.sh]: `Scaleway Technology` -> `Scaleway Cloud`

File: patches/usr/local/bin/configure_ssl.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ ! -d /etc/apache2/ssl ];
+then
+	mkdir -p /etc/apache2/ssl
```
Comment (moul) [inline patches/usr/local/bin/configure_ssl.sh]: `mkdir -p` already does an implicit `test -d`, you can remove the test to keep this script lighter

File: patches/etc/apache2/sites-available/default-ssl.conf [hunk 3]
```diff
@@ -0,0 +1,138 @@
+<IfModule mod_ssl.c>
+	<VirtualHost _default_:443>
+		ServerAdmin webmaster@localhost
+		ServerName 212.47.251.205
```
Comment (moul) [inline patches/etc/apache2/sites-available/default-ssl.conf]: This should not be hardcoded

Comment (QuentinPerez) [inline patches/etc/apache2/sites-available/default-ssl.conf]: Yep :)

**Comment by moul** [comment]
LGTM

**Comment by aimxhaisse** [comment]
LGTM

**Pull request merged**